### ISSUE TITLE
Issue installing with pip

### ISSUE THREAD

**Issue by aarbrown** [jorgebastida/awslogs#20]
Hello!  I cannot get your script (version 0.1.0) to install using pip.  There appears to be a dependency conflict issue on botocore.  Here's what I get when I try to install:

```
ᐅ pip install awslogs
Collecting awslogs
  Using cached awslogs-0.1.0.tar.gz
Collecting jmespath==0.7.1 (from awslogs)
  Using cached jmespath-0.7.1-py2.py3-none-any.whl
Collecting botocore<=1.2.3 (from awslogs)
  Using cached botocore-1.2.3-py2.py3-none-any.whl
Collecting boto3>1.0.0 (from awslogs)
  Using cached boto3-1.2.1-py2.py3-none-any.whl
Collecting termcolor>=1.1.0 (from awslogs)
  Using cached termcolor-1.1.0.tar.gz
Collecting python-dateutil>=2.4.0 (from awslogs)
  Using cached python_dateutil-2.4.2-py2.py3-none-any.whl
Collecting docutils>=0.10 (from botocore<=1.2.3->awslogs)
  Using cached docutils-0.12-py3-none-any.whl
Collecting six>=1.5 (from python-dateutil>=2.4.0->awslogs)
  Using cached six-1.10.0-py2.py3-none-any.whl
Installing collected packages: six, docutils, python-dateutil, termcolor, boto3, botocore, jmespath, awslogs



  Running setup.py install for termcolor


  Found existing installation: jmespath 0.9.0
    Uninstalling jmespath-0.9.0:
      Successfully uninstalled jmespath-0.9.0

  Running setup.py install for awslogs
    Installing awslogs script to /Users/abrown/.virtualenvs/consilio/bin
Successfully installed awslogs-0.1.0 boto3-1.2.1 botocore-1.2.3 docutils-0.12 jmespath-0.7.1 python-dateutil-2.4.2 six-1.10.0 termcolor-1.1.0

ᐅ awslogs
Traceback (most recent call last):
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 612, in _build_master
    ws.require(__requires__)
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 918, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 810, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (botocore 1.2.3 (/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages), Requirement.parse('botocore<1.4.0,>=1.3.0'), {'boto3'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/abrown/.virtualenvs/python34/bin/awslogs", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 3018, in <module>
    working_set = WorkingSet._build_master()
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 614, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 627, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 810, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (botocore 1.2.3 (/Users/abrown/.virtualenvs/python34/lib/python3.4/site-packages), Requirement.parse('botocore<1.4.0,>=1.3.0'), {'boto3'})
```

**Comment by vlcinsky** [comment]
I have the same issue.

The best would be to work with latest versions of required python packages, but if we have to stay with proven version, than it would be practical using `requirements.txt` with exact versions which are working well (or using fixed versions in `setup.py`.

**Comment by jorgebastida** [comment]
Just released ``0.1.1``. It should be fine now that boto3 dependencies has been fixed. https://pypi.python.org/pypi/awslogs/

**Comment by vlcinsky** [comment]
@jorgebastida Thanks
I confirm, that the installation works well now.
